cluster/ubuntu/build.sh

```diff
@@ -40,7 +40,7 @@ cp flannel-${FLANNEL_VERSION}/flanneld binaries/minion
 
 # ectd
 echo "Download etcd release ..."
-ETCD_VERSION="v2.0.9"
+ETCD_VERSION="v2.0.12"
```

Comment:
Oh, I apologize, I thought we backed that out.

On Mon, Jun 29, 2015 at 10:27 AM, Zach Loafman <notifications@github.com>
wrote:

> In cluster/ubuntu/build.sh
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/10393#discussion_r33488846>
> :
>
> > @@ -40,7 +40,7 @@ cp flannel-${FLANNEL_VERSION}/flanneld binaries/minion
> >
> >  # ectd
> >  echo "Download etcd release ..."
> > -ETCD_VERSION="v2.0.9"
> > +ETCD_VERSION="v2.0.12"
>
> We've been running 2.0.12 for a week now: #10140
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/10140>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/10393/files#r33488846>
> .
>